vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go

```diff
@@ -40,6 +40,7 @@ type VerticalPodAutoscalerList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:storageversion
 // +kubebuilder:resource:shortName=vpa
+// +kubebuilder:subresource:status
```

Comment:
I think we need the same kubebuilder annotation for the `v1beta2` type, otherwise the next execution of `hack/generate-crd-yaml.sh` will remove the `status` subresource again from that API version.

Reply:
done